TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -72,20 +71,20 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
                     parentFragmentManager.beginTransaction()
                         .replace(
                             R.id.dual_screen_end_container_id,
-                            NoteFragment(), null
+                            NoteFragment.newInstance("test1"), null
```

Comment:
should "test1" be hardcoded?

Reply:
"test1" is a placeholder for now. when we get the note model created, we'll want to pass this in instead (or some kind of identifier that helps us find the correct file based on the item selected in the list)